temp/legacy-code/src/main/java/com/ivy/legacy/domain/deprecated/logic/AccountCreator.kt

```diff
@@ -54,7 +54,8 @@ class AccountCreator @Inject constructor(
                 icon = data.icon,
                 includeInBalance = data.includeBalance,
                 orderNum = accountDao.findMaxOrderNum().nextOrderNum(),
-                isSynced = false
+                isSynced = false,
+                id = account.id.value
```

Comment:
Fckin default values caused this bug 😀 You see now why I'm banning them with Detekt. Thanks for fixing! Respect.

Reply:
Yes, now I get to know why default values cause so many problems xD